improved the full example section (#141)
trustoverip/tswg-did-method-webs-specification: spec/diddocuments.md

1. `did:webs` DID documents MUST be pure JSON. They MAY be processed as JSON-LD by prepending an `@context` if consumers of the documents wish.
1. All hashes, cryptographic keys, and signatures MUST be represented as [[ref: CESR]] strings. This is an approach similar to multibase, making them self-describing and terse.

> To better understand the cryptographic verifiable data structures used, see the implementors guide description of the [[def: KERI event stream chain of custody]]

Reviewer: ```suggestion
To better understand the cryptographically verifiable data structures used, see the implementors guide description of the [[def: KERI event stream chain of custody]]
```

> To understand the KERI AID commands, resulting [[ref: KERI Event Stream]], and the corresponding `did:webs` DID document see the original [[ref: did:webs Reference Implementation]] GETTING STARTED guide.

Reviewer: ```suggestion
To understand the KERI AID commands resulting in the [[ref: KERI Event Stream]] and the corresponding `did:webs` DID document see the original [[ref: did:webs Reference Implementation]] [GETTING STARTED guide](https://github.com/hyperledger-labs/did-webs-resolver/blob/main/GETTING_STARTED.md).
```

### Full Example
This section is informative.

> To walk through a real-world example, please see the GETTING STARTED example of the original [[ref: did:webs Reference Implementation]] as it walks users through many did:webs related examples (and associated KERI commands) to show how they work together.

Reviewer: ```suggestion
To walk through a real-world example, please see the GETTING STARTED guide in the [[ref: did:webs Reference Implementation]] as it walks users through many did:webs related tasks (and associated KERI commands) to demonstrate how they work together.
```

The following blocks contain fully annotated examples of a KERI AID with two events, an [[ref: inception event]] and an [[ref: interaction event]].
* The [[ref: inception event]] designates some [[ref: witnesses]] in the `b` field.
* The [[ref: inception event]] designates multiple public signing keys in the `k` field.
* The [[ref: inception event]] designates multiple rotation keys in the `n` field.
* The [[ref: interaction event]] cryptographically anchors data associated with the SAID `EoLNCdag8PlHpsIwzbwe7uVNcPE1mTr-e1o9nCIDPWgM`.
* The reply 'rpy' events specify an Agent endpoint, etc.

> After the below KERI Event Stream we show the resulting DID document that an implementation would generate assuming the implementation was running on the `example.com` domain with no unique port and no additional path defined:

Reviewer: ```suggestion
Below, we show the KERI Event Stream that will be associated with the resulting generated DID document. These documents were generated for the `example.com` domain with no associated port or additional path defined:
```
